setup.py

```diff
@@ -13,36 +13,52 @@
 def install_configs():
     """ Install configuration files to /etc """
 
-    dst_ryu_conf_dir = '/etc/ryu/'
+    dst_ryu_conf_dir = '/etc/faucet/'
     dst_ryu_conf = os.path.join(dst_ryu_conf_dir, 'ryu.conf')
-    dst_faucet_conf_dir = '/etc/ryu/faucet/'
-    src_ryu_conf = resource_filename(__name__, "etc/ryu/ryu.conf")
-    src_faucet_conf_dir = resource_filename(__name__, "etc/ryu/faucet/")
-    faucet_log_dir = '/var/log/ryu/faucet/'
+    dst_faucet_conf_dir = '/etc/faucet/'
+    src_ryu_conf = resource_filename(__name__, "etc/faucet/ryu.conf")
+    src_faucet_conf_dir = resource_filename(__name__, "etc/faucet/")
+    faucet_log_dir = '/var/log/faucet/'
+
+    old_dst_ryu_conf_dir = '/etc/ryu/'
+    old_dst_ryu_conf = os.path.join(dst_ryu_conf_dir, 'ryu.conf')
+    old_dst_faucet_conf_dir = '/etc/ryu/faucet/'
+    old_src_ryu_conf = '/etc/ryu/ryu.conf'
+    old_src_faucet_conf_dir = '/etc/ryu/faucet/'
+    old_faucet_log_dir = '/var/log/ryu/faucet/'
 
     try:
         if not os.path.exists(dst_ryu_conf_dir):
             print("Creating %s" % dst_ryu_conf_dir)
             os.makedirs(dst_ryu_conf_dir)
         if not os.path.isfile(dst_ryu_conf):
-            print("Copying %s to %s" % (src_ryu_conf, dst_ryu_conf))
-            shutil.copy(src_ryu_conf, dst_ryu_conf)
+            if os.path.exists(old_src_ryu_conf) and os.path.isfile(old_src_ryu_conf):
+                print("Copying %s to %s" % (old_src_ryu_conf, dst_ryu_conf))
+                shutil.copy(old_src_ryu_conf, dst_ryu_conf)
+            else:
+                print("Copying %s to %s" % (src_ryu_conf, dst_ryu_conf))
+                shutil.copy(src_ryu_conf, dst_ryu_conf)
         if not os.path.exists(dst_faucet_conf_dir):
             print("Creating %s" % dst_faucet_conf_dir)
             os.makedirs(dst_faucet_conf_dir)
         for file_name in os.listdir(src_faucet_conf_dir):
             src_file = os.path.join(src_faucet_conf_dir, file_name)
             dst_file = os.path.join(dst_faucet_conf_dir, file_name)
-            if os.path.isfile(src_file) and not os.path.isfile(dst_file):
+            alt_src = os.path.join(old_src_faucet_conf_dir, file_name)
+            if os.path.isfile(dst_file):
+                continue
+            elif os.path.isfile(alt_src):
+                print("Copying %s to %s" % (alt_src, dst_file))
+                shutil.copy(alt_src, dst_file)
```

Comment:
This code doesn't seem to work when running under a virtualenv, steps to reproduce:

```
virtualenv -p python3 /tmp/testenv
source /tmp/testenv/bin/activate
mkdir -p /tmp/testenv/etc/ryu/faucet
echo "test" > /tmp/testenv/etc/ryu/faucet/faucet.yaml
pip3 install --upgrade --pre --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple faucet
```

What we would expect to see in `/tmp/testenv/etc/faucet/faucet.yaml`is "test" instead we see the sample configuration file.